sdk/src/connections/target/addi9036_sensor.cpp

```diff
@@ -394,11 +375,9 @@ Addi9036Sensor::setFrameType(const aditof::FrameDetails &details) {
         /* Set the frame format in the driver */
         CLEAR(fmt);
         fmt.type = dev->videoBuffersType;
-        
-#if defined TOYBRICK
+        //#if defined TOYBRICK //????
```

Comment:
The following line was causing an error on other boards. 
Will test again.

Reply:
@acozma  was thinking that it might work. We should test, yes.

Reply:
turns out it does not. will add the macro back

Reply:
ok. On which platform it did not work?